server/src/db/schema.sql

```diff
@@ -39,9 +39,3 @@ CREATE TABLE IF NOT EXISTS words (
   word VARCHAR(100) NOT NULL,
   created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 );
-
```

Comment:
deleted just cus the game is not usable with no csv file or image, on seed-db just run postman requests to make games instead